tc-tests-utils.sh

```diff
@@ -413,6 +413,21 @@ do_deepspeech_binary_build()
     deepspeech
 }
 
+do_deepspeech_ndk_build()
+{
+  arch_abi=$1
+
+  cd ${DS_DSDIR}/native_client/
+
+  ${ANDROID_NDK_HOME}/ndk-build \
```

Comment:
hm I can likely move that into makefile to be consistent with the rest

Reply:
Or not, since `ndk-build` is really too different than the `gcc` based targets we have in `Makefile`. I could still do a specific target?